src/server/controllers/incidents.js

```diff
@@ -201,7 +202,15 @@ module.exports = {
   },
 
   // get all incidents
-  list(req, res) {
+  async list(req, res) {
+    if (res.locals.roleId === 2) {
+      const includeForAssignee = listAssigneeIncidentsIncludes();
+      const findIncidents = await User.findOne({ where: { id: res.locals.id}, include: includeForAssignee});
+      const userAssignedIncidents = findIncidents.assignedIncidents.map(oneIncident => oneIncident);
```

Comment:
This map is not necessary and it's not doing anything. You are just returning the same thing. It's like mapping an array and returning the same array.